packages/frontend/src/hooks/useProjectAccess.tsx

```diff
@@ -121,11 +120,10 @@ export const useUpdateProjectAccessMutation = (projectUuid: string) => {
                     title: 'Updated project access role',
                 });
             },
-            onError: async (error1) => {
-                const [title, ...rest] = error1.error.message.split('\n');
-                showToastError({
-                    title,
-                    subtitle: rest.join('\n'),
+            onError: async ({ error }) => {
+                showToastApiError({
+                    title: 'Failed to update project access role',
+                    apiError: error,
```

Comment:
TODO for me: check this one locally

Reply:
all good 
<img width="496" alt="image" src="https://github.com/lightdash/lightdash/assets/7611706/978f4594-1e90-4bef-a90d-c128c30d89c6">